components/accelerator/nvidia/infiniband/component_output.go

```diff
@@ -46,6 +49,9 @@ type Output struct {
 	InfinibandClassExists bool                    `json:"infiniband_class_exists"`
 	IbstatExists          bool                    `json:"ibstat_exists"`
 	Ibstat                infiniband.IbstatOutput `json:"ibstat"`
+
+	MellanoxPCIDevices       pci.Devices `json:"mellanox_pci_devices"`
```

Comment:
should this be part of ib? i am not sure.

Reply:
Moved to new PCI component